x-pack/plugins/security_solution/public/management/pages/policy/view/tabs/policy_tabs.tsx

```diff
@@ -98,7 +144,14 @@ export const PolicyTabs = React.memo(() => {
         content: (
           <>
             <EuiSpacer />
-            <PolicyTrustedAppsLayout />
+            <PolicyArtifactsLayout
+              policyItem={policyItem}
+              labels={trustedAppsLabels}
+              getExceptionsListApiClient={() => TrustedAppsApiClient.getInstance(http)}
+              searcheableFields={[...TRUSTED_APPS_SEARCHABLE_FIELDS]}
```

Comment:
should these be memoized in order to avoid unnecessary re-renders?

(same comment below)

Reply:
I don't think so, this is from a static import. I'm destructuring it because it's a readonly type. I can just define it as readonly on the component itself.

Reply:
Ok. Just realize that because you are passing a new function everytime to `getExceptionsListApiClient` and a new Array to `searcheableFields`, the component will always trigger a re-render (props strict equality ( `===` ) will report `false`)

Reply:
oh yes, we can use `useCallback` for the `getExceptionsListApiClient` function

Reply:
and maybe use instead of creating a new array and spreading the values, just cast the type?

Reply:
the searchableFields destructured array is fixed in this commit https://github.com/elastic/kibana/pull/126685/commits/d99ba65dbe76e128cc86b5ba44961b30fc81d8c8

Reply:
About the `getExceptionsListApiClient`, all of this code is inside a `useMemo`, do you think it's necessary to have functions outside using `useCallback`?